atum-s3-sdk-extension/pom.xml

```diff
@@ -47,6 +47,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>za.co.absa.commons</groupId>
+            <artifactId>commons_${scala.binary.version}</artifactId>
+            <version>${commons.version}</version>
```

Comment:
Last question. Does this need to be here if there are commons already in atum?

Reply:
`Regional.scala` in this package needs _S3Location_ stuff from _Absa commons_.

It would work with the dependency on a transitive basis, too (so it would not need to be listed to work), but since we are explicitly using it in this package, we should also explicitly list it, IMHO.